tests/integration/aof-multi-part.tcl

```diff
@@ -930,33 +923,33 @@ tags {"external:skip"} {
 
             # We can see four history AOFs (Evolved from two BASE and two INCR)
             assert_aof_manifest_content $aof_manifest_file {
-                {file appendonly.aof.7.base.rdb seq 7 type b}
-                {file appendonly.aof.2.incr.aof seq 2 type h}
-                {file appendonly.aof.6.base.rdb seq 6 type h}
-                {file appendonly.aof.1.incr.aof seq 1 type h}
-                {file appendonly.aof.5.base.rdb seq 5 type h}
-                {file appendonly.aof.3.incr.aof seq 3 type i}
+                {file appendonly.aof.5.base.rdb seq 5 type b}
+                {file appendonly.aof.7.incr.aof seq 7 type h}
+                {file appendonly.aof.4.base.rdb seq 4 type h}
+                {file appendonly.aof.6.incr.aof seq 6 type h}
+                {file appendonly.aof.3.base.rdb seq 3 type h}
+                {file appendonly.aof.8.incr.aof seq 8 type i}
```

Comment:
can you explain this change?
i expected the base file indexes to be incremented by one? but it looks like some here got decremented.

Reply:
? 3 and 4 is history file.